README.md

```diff
@@ -55,19 +69,13 @@ def augment_data(images, labels):
   inputs = {"images": images, "labels": labels}
   outputs = augmenter(inputs)
   return outputs['images'], outputs['labels']
-```
-
-Augment a `tf.data.Dataset`:
 
-```python
+# Augment a `tf.data.Dataset`
 dataset = tfds.load('rock_paper_scissors', as_supervised=True, split='train')
 dataset = dataset.batch(64)
```

Comment:
re: OOM

Perhaps we should just decrease batch size here if this is crashing a Colab.

Reply:
We could also use a smaller model like MobileNetV3Small, althought DenseNet121 is already a relatively small backbone in terms of # of params.